inst/include/interface/rcpp/rcpp_objects/rcpp_interface_base.hpp

```diff
@@ -225,31 +292,67 @@ class ParameterVector{
      * @param value The value to be assigned
      */
     void fill_max(double value){
-        for(int i = 0; i < this->storage_m.size(); i++){
-            Parameter p = Rcpp::as<Parameter>(this->storage_m[i]);
-            p.max_m = value;
-            this->storage_m[i] = Rcpp::wrap(p);
+        for (size_t i = 0; i < this->storage_m->size(); i++) {
+            storage_m->at(i).max_m = value;
+        }
+    }
+
+    /**
+     * @brief Printing methods for ParameterVector
+     *
+     */
+    void show() {
+        std::cout << this->storage_m->data() << "\n";
+
+        for (size_t i = 0; i < this->storage_m->size(); i++) {
+            std::cout << storage_m->at(i) << "  ";
         }
     }
 
 };
 uint32_t ParameterVector::id_g = 0;
 
+std::ostream& operator<<(std::ostream& out, ParameterVector& v) {
+    out << "[";
+    size_t size = v.size();
+    for (size_t i = 0; i < size - 1; i++) {
+        out << v[i] << ", ";
+    }
+    out << v[size - 1] << "]";
+    return out;
+}
 
 /**
  *@brief Base class for all interface objects
  */
 class FIMSRcppInterfaceBase {
- public:
-  /**< FIMS interface object vectors */
-  static std::vector<FIMSRcppInterfaceBase *> fims_interface_objects;
+public:
+
+    bool finalized = false;
+
+    /**< FIMS interface object vectors */
+    static std::vector<FIMSRcppInterfaceBase *> fims_interface_objects; /* TODO: document*/
```

Comment:
@msupernaw nothing that I am trying here is working, note that I know that /* TODO: ... should be /**< TODO */ but even that is not working. I also tried removing the comment on line 333 just in case that was messing things up. HELP

Reply:
We got it! Thank you 🙏